RiotSharp/LeagueEndpoint/LeaguePosition.cs

```diff
@@ -5,38 +5,60 @@ namespace RiotSharp.LeagueEndpoint
     /// <summary>
     /// Team or summoner in a league (League API).
     /// </summary>
-    public class LeagueEntry
+    public class LeaguePosition
     {
-        internal LeagueEntry() { }
+        internal LeaguePosition() { }
 
         /// <summary>
+        /// The name of the league of the participant.
+        /// Only when it's called from the GetLeaguePositions()
+        /// </summary>
+        [JsonProperty("leagueName")]
+        public string LeagueName { get; set; }
+
+        /// <summary>
+        /// The queue type of the league.
+        /// Only for the GetLeaguePositions() -> don't exist when it's an entry from a League
+        /// as there is already the Queue property in this case.
+        /// </summary>
+        [JsonProperty("queueType")]
+        public string QueueType { get; set; }
+        
+        /// <summary>
         /// The league division of the participant.
         /// </summary>
-        [JsonProperty("division")]
+        [JsonProperty("rank")]
         public string Division { get; set; }
```

Comment:
let's rename the property to `Rank` too

Reply:
I had let like this because people are used to `Division` but will change this too.